app/serializers/data_explorer/small_badge_serializer.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
 class DataExplorer::SmallBadgeSerializer < ApplicationSerializer
-  attributes :id, :name, :badge_type, :description, :icon
+  attributes :id, :name, :display_name, :badge_type, :description, :icon
+
+  def display_name
+    object&.name
```

Comment:
Isn't `display_name` available? What's the type of `object`?

Reply:
OMG 🤦. I had a false feeling that it could be a super-fast fix. `display_name` is available indeed. I've fixed it and checked it with a changed locale.